### ISSUE TITLE
Inconsistent behavior of input().csv().line()

### ISSUE THREAD

**Issue by axiopost** [vectorgraphics/asymptote#186]
When loading a csv file with imput().csv().line(), empty lines are ignored if they are at the beginning of the file, but return a 0 value in the middle of the file.

Example with this file [sample.txt](https://github.com/vectorgraphics/asymptote/files/5505297/sample.txt)
```
#First line commented

1,2,3

4,5,6

#End of file
```

This code 
```
write("Asymptote version ",VERSION);
int[][] a=input("sample.txt").csv().line();
write(a);
```
results in:
```
Asymptote version 2.67
1	2	3
0
4	5	6
0
```

Also it would be useful to be able to provide a data separator other than the comma. Maybe with something like `input("sample.txt").csv(";").line();`

**Comment by axiopost** [comment]
In addition, during import, it would make more sense if the empty fields in the file gives undefined values instead of forcing them to 0.
Example applying `int[][] a=input("sample.txt").csv().line(); write(a); `to this file:
`1,,1`
Result in:
```
1	0	1
```
instead of:
```
1		1
```

**Comment by johncbowman** [comment]
Fixed in f513ffde5c8540e1b45a93fd0391541a6f4aa2f7.
